src/modules/helpers/index.ts

```
@@ -797,6 +797,25 @@ export class HelpersModule {
     return object[key];
   }
 
+  /**
+   * Returns a random [key, value] pair from given object or `undefined` if no key could be found.
```

for consistency with the other object methods, it should throw if no key could be found, not return undefined

---

https://github.com/faker-js/faker/pull/2123/files#diff-3f6b4fc424ba2cedcbf5f612ba256d248f0c05f46314eed1dfa81590bcd429fdR767

Docs for objectKey and objectValue also mention it in the same way, hence maybe I was inclined towards doing that.

Do you want me to create a new PR to fix those as well?

---

Yea that would be good. Docs are definitely wrong for that.

---

I have made those changes within this PR only [here](https://github.com/faker-js/faker/pull/2123/files#diff-3f6b4fc424ba2cedcbf5f612ba256d248f0c05f46314eed1dfa81590bcd429fdR767).
Hope this is OK. If not, I will create a new one.